wallet/src/actors/app/handlers/validate_mnemonics.rs

```diff
@@ -14,7 +12,8 @@ pub struct ValidateMnemonicsRequest {
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct ValidateMnemonicsResponse {
-    pub valid: bool,
+    pub exist: bool,
+    pub wallet_id: String,
```

Comment:
Breaking change. What's the equivalent PR in sheikah?

Reply:
I think they were not using `validate_mnemonics` until now, so they shouldn't have any problem. :smile:

Reply:
I found it here, it is used when importing mnemonics:

https://github.com/witnet/sheikah/blob/ff65864a464591ef3275b95caac82465fb867137/src/store/wallet.js#L773